build.ps1

```diff
@@ -60,6 +60,31 @@ function NugetPack {
     }
 }
 
+function PackSelfContainProject {
+    param($assemblyFolder, $nuspecPath)
+
+    $nuspecFile = Get-Item $nuspecPath
+    $nuspecName = $nuspecFile.Name
+    $nuspecFolder = $nuspecFile.Directory.FullName
+    $nuspecFolderName = $nuspecFile.Directory.Name
+    $targetFolder = "PACK\$nuspecFolderName"
+
+    if (Test-Path $targetFolder)
+    {
+        $null = Remove-Item $targetFolder -Force -Recurse
+    }
+    $null = New-Item -ItemType Directory -Force -Path $targetFolder
+    $null = New-Item -ItemType Directory -Force -Path "$targetFolder\tools\"
+
+    Copy-Item -Path "$nuspecFolder\**" -Destination "$targetFolder" -Force -Recurse
+    Copy-Item -Path "$assemblyFolder\*.dll" -Destination "$targetFolder\tools\" -Force
+    Copy-Item -Path "$assemblyFolder\*.exe" -Destination "$targetFolder\tools\" -Force
+    Copy-Item -Path "$assemblyFolder\*.exe.config" -Destination "$targetFolder\tools\" -Force
+
+    & $nuget pack "$targetFolder\$nuspecName" -Version $version -OutputDirectory artifacts\$configuration
```

Comment:
yes, $version is a global variable

---
In reply to: [95950135](https://github.com/dotnet/docfx/pull/1167#discussion_r95950135) [](ancestors = 95950135)